images/cilium/Dockerfile.dockerignore

```
@@ -28,4 +28,5 @@
 !/pkg
 !/plugins/cilium-cni
 !/proxylib
+!/operator
```

As per my [previous comment](https://github.com/cilium/cilium/pull/14069/files#r526768402):

```suggestion
!/operator/option
```

---

Also, please keep the list alphabetically sorted.

---

thanks, my bad :face_with_thermometer: